client/trino-cli/src/main/java/io/trino/cli/Console.java

```diff
@@ -100,6 +100,7 @@ public Integer call()
         return run() ? 0 : 1;
     }
 
+    @SuppressWarnings("CheckReturnValue")
```

Comment:
this suppresses all such checks on a method that's longer than few lines. apply suppression more selectively